pkg/render/fluentd.go

```diff
@@ -166,6 +167,7 @@ type FluentdConfiguration struct {
 	UsePSP bool
 	// Whether to use User provided certificate or not.
 	UseSyslogCertificate bool
+	EnterpriseVariant    bool
```

Comment:
This information is already available via the `InstallationSpec` that is provided - please use that instead of adding another config option.